src/index.css

```diff
@@ -72,3 +80,19 @@
     content: none !important;
   }
 }
+
+:global(.relative) {
```

Comment:
@mike-robertson may have some comments to make about the use of global classes here.